src/components/header/header.tsx

```diff
@@ -136,15 +171,45 @@ const Header: React.FC<HeaderProps> = (props) => {
         <HeaderSwoosh />
         {props.siteTitle}
       </SiteTitle>
-      <SiteMenu
-        aria-label="Open menu"
-        $lightTheme={Boolean(!isHeaderTransparent && props.$lightTheme)}
-        onClick={() => {
-          setIsNavigationVisible(!isNavigationVisible);
-        }}
-      >
-        {!isNavigationVisible ? <BurgerMenu /> : <CloseBurgerMenuIcon />}
-      </SiteMenu>
+      <div>
+        {props.translation
+          ? languages.map((lng) => {
+              return (
+                <StyledLanguageLink
+                  key={lng}
+                  to={language === lng ? '' : props.translation || ''}
+                  language={lng}
+                  selected={language === lng}
+                >
+                  {lng}
+                </StyledLanguageLink>
+              );
+            })
+          : props.showLanguageSwitch &&
```

Comment:
I don't get the logic here :D 

First, you check if there are `languages`. If true, you iterate over them. If false, you check for `showLanguageSwitch`. If true, you again map over `languages`.

I guess the this can be simplified to:
```js
(props.showLanguageSwitch && props.translation) ? ... : ...
```